developer_manual/app_publishing_maintenance/app_upgrade_guide/upgrade_to_25.rst

```diff
@@ -28,12 +28,29 @@ See `Github issue <https://github.com/nextcloud/server/issues/32060>`_.
 Front-end changes
 -----------------
 
-tbd
+Disabled keyboard shortcuts
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+A global option to disable keyboard shortcuts was added to the accessibility settings.
+Since it heavily depends on the screenreader and tools that you use if Ctrl and/or Alt or other things are okay to use
+or not and maintaining a more detailed list is too much effort, we went for a global on/off switch. Apps can use this
+public javascript API call to determine whether the user used the opt-out: ``OCP.Accessibility.disableKeyboardShortcuts()``.
+If that is the case, no additional shortcuts shall be registered by any app. Only ``space`` to toggle checkboxes and
+``enter`` to submit the currently active buttons/links are okay to be used.
+See `Github issue <https://github.com/nextcloud/server/pull/34081>`_.
 
 Back-end changes
 ----------------
 
-tbd
+christophwurst/nextcloud replaced
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The `christophwurst/nextcloud <https://packagist.org/packages/christophwurst/nextcloud>`_ composer package was replaced
+with a now Nextcloud owned `nextcloud/ocp <https://packagist.org/packages/nextcloud/ocp>`_ package. The content is the
+same and all older versions were generated, so you can transition right away no matter which versions you support.
+We also have a GitHub Actions cron job template available at
+`https://github.com/nextcloud/.github/blob/master/workflow-templates/update-nextcloud-ocp.yml <https://github.com/nextcloud/.github/blob/master/workflow-templates/update-nextcloud-ocp.yml>`_
+which updates the packages every sunday, so you can make sure your app is still compatible with the latest OCP.
```

Comment:
this info is not relevant for the upgrade and can go into the CI docs

Reply:
Will update CI docs in a second PR to not block this file or get conflicts when others document their 25 stuff

Reply:
Done in https://github.com/nextcloud/documentation/pull/9299